components/notification-controller/base/service-account.yaml

```diff
@@ -1,37 +0,0 @@
----
-apiVersion: v1
-kind: ServiceAccount
```

Comment:
With this PR, do we still have the configs in this file? Same goes for the namespace configs.